presto-hive/src/test/java/com/facebook/presto/hive/metastore/thrift/InMemoryHiveMetastore.java

```diff
@@ -572,6 +575,16 @@ private static boolean isParentDir(File directory, File baseDirectory)
         return false;
     }
 
+    private static void deleteDirectory(File dir)
```

Comment:
`InMemoryHiveMetastore` sounds like "keep things in memory", not on disk. Maybe, this is actually so because metadata is in memory and data is on disk, which wouldn't be sad.

Reply:
This is the weird thing about the Hive Metastore. I touches the data "almost" never. The metastore does that only for `DROP TABLE` and `DROP PARTITION` operations. It removes the managed data. In `InMemoryHiveMetastore` we want to simulate the same semantics.